x-pack/plugins/security_solution/common/detection_engine/rule_monitoring/api/get_rule_execution_events/request_schema.ts

```diff
@@ -41,3 +40,30 @@ export const GetRuleExecutionEventsRequestQuery = t.exact(
     per_page: DefaultPerPage, // defaults to 20
   })
 );
+
+type SortOrder = t.TypeOf<typeof SortOrder>;
+const SortOrder = t.keyof({ asc: null, desc: null });
```

Comment:
@oatkiller This type doesn't exclusively belong to the `rule_monitoring` subdomain, it's used from some other places in the code too, for instance, from the rules `_find` endpoint which is part of `rule_management`.

I think it should be either in some "common Detections APIs types and code" folder, or in `rule_management` if it's only used from the endpoints related to rule management and monitoring.

@xcrzx What do you think considering our upcoming adoption of the OpenAPI tooling? Should we have a sort of "common folder for the API stuff" that @elastic/security-detection-rule-management would own?

Reply:
Thanks for the reply. I closed this per our other discussion. I'm going to focus my efforts on assigning owners to Cypress tests for now.